tests/system/lab/test

```diff
@@ -0,0 +1,580 @@
+#!/bin/bash
```

Comment:
We should move this to a python script and run with pytest:

1) have a test class with different tests, setup and teardown
2) use requests to replace curl and better loop/ retries handling
3) add assertions and have better junit reports and pass/ fail reports.

But first goal is to test (CI and individually) this setup on new RPi0+ ePHAT setup as is; and rely in exit code.